### ISSUE TITLE
gnt-instance info does not show instance tags

### ISSUE THREAD

**Issue by candlerb** [ganeti/ganeti#1302]
`gnt-cluster info` shows cluster tags, but `gnt-instance info` does not show instance tags, and I think it would be nice if it did.

At the moment you have to explicitly ask for tags using a different command, e.g.

```
gnt-instance list -o name,tags [<instance>]
```

**Comment by candlerb** [comment]
Maybe you misunderstood the problem description.

The problem is that *instance tags* are not shown by `gnt-instance info`, and it's definitely still the case with `ganeti 2.16.0~rc2` from Ubuntu 18.04, as I've just retested it.

Here's an example of an instance with a tag:

```
root@ix-infra1:~# gnt-instance list -o name,tags ix-dc01
Instance                Tags
ix-dc01.int.example.net service:ad
root@ix-infra1:~# gnt-instance info ix-dc01 | grep service
root@ix-infra1:~#
```

You can see that the tag is not returned by `gnt-instance info`.

Version:

```
root@ix-infra1:~# gnt-cluster version
Software version: 2.16.0~rc2
Internode protocol: 2160000
Configuration format: 2160000
OS api version: 20
Export interface: 0
VCS version: (ganeti) version 2.16.0~rc2-1build1
root@ix-infra1:~# 
```

**Comment by apoikos** [comment]
There's a separate command, `gnt-instance list-tags` for that, and of course tags are available through the `gnt-instance list` interface when asked for explicitly. I don't think this is a bug, but if it helps, it wouldn't harm to add the output to `gnt-instance info`.

**Comment by candlerb** [comment]
Yes, it's very minor.  There is a `gnt-cluster list-tags` command, but `gnt-cluster info` shows the tags too.  So I was expecting `gnt-instance info` to be the same, and tell me everything about the instance.

Having said that: I've just tested it with `gnt-node`, and I find that `gnt-node list-tags` shows the tags, but `gnt-node info` does not.
